manifest/package.xml

```diff
@@ -92,6 +97,14 @@
 		<members>Session__c.All</members>
 		<name>ListView</name>
 	</types>
+	<types>
```

Comment:
@t-gul one of the matching rule types can be removed I think :/ One of them should work properly. Can you test and leave the one that works?

Reply:
both of them work, I'm erase one of them